buildkitd/entrypoint.sh

```diff
@@ -248,5 +260,16 @@ do
         fi
         exit "$code"
     fi
+
+    for PID in $(pgrep -P 1)
+    do
+        # Sometimes, child processes can be reparented to the init (this script). One
+        # common instance is when something is OOM killed, for instance. This enumerates
+        # all those PIDs, and kills them to prevent accidential "ghost" loads.
+        if [ "$PID" != "$execpid" ]; then
+            ! ${BUILDKIT_DEBUG} || echo "$(date) | $PID($(cat /proc/$PID/cmdline)) killed" >> /var/log/oom_adj
```

Comment:
Nit: Style.

```suggestion
            ! "${BUILDKIT_DEBUG}" || echo "$(date) | $PID($(cat /proc/$PID/cmdline)) killed" >> /var/log/oom_adj
```

Reply:
I need to tweak this more, yes, partly a copy paste error